README.md

```diff
@@ -15,45 +17,89 @@ The code consists of a patch series morphed and rebased onto every new Chrome re
+NaïveProxy uses Chrome's network stack. Thus, the traffic behavior intercepted by the censor is identical to regular HTTP/2 traffic between Chrome and standard Frontend (e.g., Caddy, HAProxy).
 
 Frontend also reroutes unauthenticated users and active probes to a backend HTTP server, making it impossible to detect the existence of a proxy, like this: Probe ⟶ Frontend ⟶ index.html
 
 Starting from v84, users can run a naïve fork of Caddy forwardproxy without the Naïve server.
 
 ## Download
 
-See [latest release](https://github.com/klzgrad/naiveproxy/releases/latest). Linux, Windows, Mac OS, and various OpenWrt targets are supported.
+The simplest, cross-platform way is to download from [GitHub Releases](https://github.com/klzgrad/naiveproxy/releases/latest). 
+
+Linux, Windows, Mac OS, and various OpenWrt targets are supported.
 
 Users should always use the latest version to keep signatures identical to Chrome.
 
-Note: On Linux libnss3 must be installed before using the prebuilt binary.
+Note: On Linux, libnss3 must be installed before using the prebuilt binary.
 
 ## Setup
 
-On the server, build and run Caddy v2 with naïve fork of forwardproxy: (Golang v1.14+ required)
+### Server-side
+
+On the server, build and run Caddy v2 with naïve fork of forwardproxy: **(Golang v1.14+ required)**
+
 ```sh
-go get -u github.com/caddyserver/xcaddy/cmd/xcaddy
-~/go/bin/xcaddy build --with github.com/caddyserver/forwardproxy@caddy2=github.com/klzgrad/forwardproxy@naive
-sudo setcap cap_net_bind_service=+ep ./caddy
+$ go get -u github.com/caddyserver/xcaddy/cmd/xcaddy
+$ ~/go/bin/xcaddy build --with github.com/caddyserver/forwardproxy@caddy2=github.com/klzgrad/forwardproxy@naive
+$ sudo setcap cap_net_bind_service=+ep ./caddy
 ```
-Then `./caddy start` with the following Caddyfile (replace the example values accordingly):
+Create `Caddyfile` at `/etc/caddy/Caddyfile`,change the `<user> `& `<pass>` accordingly.
+
 ```
 :443, example.com
 tls me@example.com
 route {
   forward_proxy {
-    basic_auth user pass
+    basic_auth <user> <pass>
     hide_ip
     hide_via
     probe_resistance
   }
   file_server { root /var/www/html }
 }
 ```
-`:443` must appear first for this Caddyfile to work. For more advanced usage consider using JSON for Caddy 2's config.
+`:443` must appear first for this Caddyfile to work. For more advanced usage consider [using JSON for Caddy 2's config](https://caddyserver.com/docs/json/).
+
+#### Run Caddy as a daemon
+
+To do so, edit `/lib/systemd/system/caddy.service ` with the following content:
+
+```
+[Unit]
+Description=Caddy
+Documentation=https://caddyserver.com/docs/
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+User=caddy
+Group=caddy
+ExecStart=/usr/bin/caddy run --environ --config /etc/caddy/Caddyfile
+ExecReload=/usr/bin/caddy reload --config /etc/caddy/Caddyfile
```

Comment:
It's problematic. Where to put the caddy binary, there is no fixed location to use here.